styles.css

```diff
@@ -1,3 +1,1317 @@
+    background-repeat: no-repeat;
+}
+.breadcrumbs-nav a[href="blog-details.html"]::before {
+    content:'';
+    position: relative;
+    right: 10px;
+    top: 2px;
+    width: 2px;
+    height: 16px;
+    background-image: url("data:image/svg+xml,%3Csvg width='3' height='16' viewBox='0 0 3 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0.5' width='2' height='16' fill='%23888B8E'/%3E%3C/svg%3E ");
+    background-repeat: no-repeat;
+}
+
+.main-heading,
+.heading {
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+.sub-heading,
+.how-to-text {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+.how-to-heading {
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: center;
+    color: #21E786;
+
+}
+
+.ot-member-pos,
+.faq-description,
+.joc-text,
+.sub-text .sub-info {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.copyright-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(1px, 1px, 1px, 1px);
+    -webkit-clip-path: inset(50%);
+    clip-path: inset(50%);
+    height: 1px;
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+    white-space: nowrap;
+}
+
+ul,
+ol {
+    list-style: none;
+}
+
+.list-term {
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.list-description {
+    font-size: 56px;
+    font-weight: 400;
+    line-height: 68px;
+    color: #fff;
+    text-shadow: 0 4px 16px #FFFFFF66;
+}
+
+
+.list-item-sub,
+.sub-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    color: #C2C3C5;
+}
+
+.joc-text-diff,
+.sub-info {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 28px;
+    text-align: center;
+    color: #F7F7F9;
+}
+
+.page-footer-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 26px;
+}
+
+.fs {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+}
+
+.sub-nav-list li a,
+.com-nav-list li a {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+    color: #C2C3C5;
+
+}
+
+.main-info,
+.price-text,
+.reply {
+    font-size: 36px;
+    font-weight: 400;
+    line-height: 44px;
+}
+.main-info {
+    display: flex;
+    flex-direction: column;
+}
+.main-info img {
+    position: relative;
```

Comment:
Instead of positioning, the` main-info` grid should be built using flexbox or grid.

Flexbox and Grid are powerful layout tools in CSS that offer greater flexibility and responsiveness compared to traditional positioning. They allow for more dynamic layouts that adapt to different screen sizes and content changes.